test/e2e/tests/settings/ipfs-ens-resolution.spec.js

```diff
@@ -10,25 +155,36 @@ describe('Settings', function () {
   it('Redirects to ENS domains when user inputs ENS into address bar', async function () {
     // Using proxy port that doesn't resolve so that the browser can error out properly
     // on the ".eth" hostname.  The proxy does too much interference with 8000.
-    const { driver } = await buildWebDriver({ proxyUrl: '127.0.0.1:8001' });
-    await driver.navigate();
-
-    // The setting defaults to "on" so we can simply enter an ENS address
-    // into the address bar and listen for address change
-    try {
-      await driver.openNewPage(ENS_NAME_URL);
-    } catch (e) {
-      // Ignore ERR_PROXY_CONNECTION_FAILED error
-      // since all we care about is getting to the correct URL
-    }
-
-    // Ensure that the redirect to ENS Domains has happened
-    await driver.wait(async () => {
-      const currentUrl = await driver.getCurrentUrl();
-      return currentUrl === ENS_DESTINATION_URL;
-    }, tinyDelayMs);
-
-    await driver.quit();
```

Comment:
no need to start or stop the driver manually, as now it's done within the fixtures